src/lib/data/nodes_desc.json

```diff
@@ -2658,37 +2653,5 @@
 	"N573": {
 		"name": "Stance Breaker",
 		"stats": ["50% reduced Enemy Chance to Block Sword Attacks"]
-	},
-	"N574": {
```

Comment:
also noted that these new nodes have been removed - you might want to rebase with main?